core/lib/libtomcrypt/sub.mk

```diff
@@ -114,6 +114,7 @@ _CFG_CRYPTO_WITH_ASN1 := $(call cryp-one-enabled, RSA DSA ECC)
 _CFG_CRYPTO_WITH_FORTUNA_PRNG := $(call cryp-all-enabled, AES SHA256)
 
 cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -DLTC_SMALL_CODE -DLTC_NO_FAST
+cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -Os
```

Comment:
Beware, I understand that  `LTC_NO_FAST` is mandatory  to avoid unaligned access exceptions? Did not notice any issue when running SHA and AES perf tests, buffers must have been naturally aligned (makes sense).